tests/input/test_inputconfigurator.py

```diff
@@ -174,7 +174,7 @@ def test_validate_yaml_validation_error(self, tmp_path):
         assert "YAML file validation error" in str(excinfo.value)
 
     # -----------------------------
-    # Tests for export_schema_to_yaml
+    # Tests for export_schema_to_yaml: file contents
     # -----------------------------
     def test_export_schema_to_yaml(self, tmp_path):
         schema = {"test": "value"}
```

Comment:
Could you also add a single line docstring describing what is being tested? For example. "Tests if a yaml file with the input schema is saved to the file system"